spec/index.bs

```diff
@@ -170,7 +170,10 @@ main dl:not(.switch) {
     padding-left: .5em;
 }
 
-/* Remove unnecessary extra margin on switch bodies */
+/* Tweak default "switch" statement layout to work better for our style. */
+dl.switch dt {
+    margin-left: -0.5em;
+}
```

Comment:
I did this because I was trying to use `switch`, but then I decided not to. But we need to fix our switch rendering anyway so I left it in.
![image](https://github.com/user-attachments/assets/904af1ed-9704-4c12-81c1-1743f7ebc03e)